src/org/opendatakit/validate/FormValidator.java

```diff
@@ -331,14 +341,37 @@ boolean stepThroughEntireForm(FormEntryModel model) throws InvalidReferenceExcep
     }
 
     public void validateAndExitWithErrorCode(String path) {
-        try {
-            validate(path);
-        } catch (Exception e) {
-            errors.error("\nException: ", e);
-            setError(true);
+        validateAndExitWithErrorCode(new String[] { path }, true);
```

Comment:
IntelliJ tells that this method isn't used anymore. I think it would be OK to remove it and avoid having to cast the string into an array/list.

Reply:
I left it there because it was marked as a `public` method and I wasn't sure what API this package expected to expose and if there were any external consumers (such as the opendatakit/aggregate server)

Reply:
If this were JavaRosa, then yes, we should keep the public methods and maybe mark them as `@Deprecated`, but in this case, I think it's safe to remove because this is a final app.

Reply:
I'll drop it then.